lib/defaultFormatter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var path = require('path');
+
+module.exports = function() {
+  return function(input) {
```

Comment:
I was thinking that the first function would accept options. So user would do something like this:

```js
var specialFormatter = require('special-formatter');
var myFormatter = specialFormatter({ smart: true, indentation: '\t' });
// ...
postcssReporter({ formatter: myFormatter });
```

It just seems weird because the defaultFormatter actually doesn't have any options now. (It *should* have the option to remove colors, though.)